modules/apps/commerce/commerce-account-service/src/main/java/com/liferay/commerce/account/internal/util/CommerceAccountHelperImpl.java

```diff
@@ -100,25 +99,11 @@ public String getAccountManagementPortletURL(
 
 	@Override
 	public long[] getCommerceAccountGroupIds(long commerceAccountId) {
-		List<AccountGroupRel> accountGroupRels =
-			_accountGroupRelLocalService.getAccountGroupRels(
-				AccountEntry.class.getName(), commerceAccountId);
-
-		if (accountGroupRels.isEmpty()) {
```

Comment:
@julschong Please try to keep these shortcuts, as they avoid unnecessary method call into `transform*`. The reason is same as I commented before.